flight/PiOS/STM32F4xx/inc/pios_ws2811_target.h

```diff
@@ -0,0 +1,63 @@
+/**
+ ******************************************************************************
+ * @addtogroup PIOS PIOS Core hardware abstraction layer
+ * @{
+ * @addtogroup PIOS_WS2811 WS2811 Functions
+ * @{
+ *
+ * @file       pios_ws2811_target.h
```

Comment:
This needs to be `STM32F4xx/inc/pios_ws2811_target.h` so it doesn't conflict with the F3 one. And _cfg.h matches our conventions (and the file contents) better?